CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * [CHANGE] Utilize separate protos for rule state and storage. Experimental ruler API will not be functional until the rollout is complete. #2226
 * [CHANGE] Frontend worker in querier now starts after all Querier module dependencies are started. This fixes issue where frontend worker started to send queries to querier before it was ready to serve them (mostly visible when using experimental blocks storage). #2246
 * [CHANGE] Lifecycler component now enters Failed state on errors, and doesn't exit the process. (Important if you're vendoring Cortex and use Lifecycler) #2251
+* [CHANGE] FIFO cache to evict entries based on used memory size #2319
```

Comment:
Given it's now backward compatible, please switch this entry to an `ENHANCEMENT` and re-phrase the message.